GUI/MainForm.cs

```diff
@@ -1068,7 +1068,7 @@ private void ExportToolStripMenuItem_Click(object sender, EventArgs e)
                     extensions = new[] { ((Sound)resource.Blocks[BlockType.DATA]).Type.ToString().ToLower() };
                     break;
                 case ResourceType.Texture:
-                    extensions = new[] { "png", "jpg", "gif", "bmp" };
+                    extensions = new[] { "png" };
```

Comment:
Losing JPG and gif support?

Reply:
Yes. I thought about it and png is the most sane format for this (and other code paths didnt support other formats).